lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Preheat"
+msgstr "F\xefrv\xe1rm"
+
+# MSG_WIZARD_HEATING c=20 r=3
+#: messages.c:119
+msgid "Preheating nozzle. Please ewait."
+msgstr "F\xefrv\xe1rmer munstycke. V\xe1nta."
+
+# MSG_NEW_FIRMWARE_PLEASE_UPGRADE c=20
+#: util.cpp:199
+msgid "Please upgrade."
+msgstr "V\xe1nligen uppgradera."
+
+# MSG_PRESS_TO_PREHEAT c=20 r=4
+#: Marlin_main.cpp:12049
+msgid "Press the knob to preheat nozzle and continue."
+msgstr "Tryck pa knappen f\xefr att f\xefrv\xe1rma munstycket och forts\xe1tta."
+
+# MSG_FS_PAUSE c=5
+#: fsensor.cpp:730
+msgid "Pause"
+msgstr "Paus"
+
+# MSG_POWER_FAILURES c=15\xe1\xe1zzz
+#: messages.c:77
+msgid "Power failures"
+msgstr "Str\xefmavbrott"
+
+# MSG_PRINT_ABORTED c=20
+#: messages.c:80
+msgid "Print aborted"
+msgstr "Utskriften avbr\xefts"
+
+# MSG_PREHEATING_TO_LOAD c=20
+#: ultralcd.cpp:2297
+msgid "Preheating to load"
+msgstr "F\xefrv\xe1rmer f\xefr laddn"
+
+#MSG_PREHEATING_TO_UNLOAD c=20
+#: ultralcd.cpp:2302
+msgid "Preheating to unload"
+msgstr "F\xefrv\xe1rmer for utmatn"
+
+# MSG_SELFTEST_PRINT_FAN_SPEED c=18
+#: ultralcd.cpp:8501
+msgid "Print fan:"
+msgstr "Utskriftsfl\xe1kt:"
+
+# MSG_CARD_MENU c=18
+#: messages.c:22
+msgid "Print from SD"
+msgstr "Skriv ut fran SD"
+
+# MSG_PRESS_KNOB c=20
+#: ultralcd.cpp:2130
+msgid "Press the knob"
+msgstr "Tryck pa knappen"
+
+# MSG_PRINT_PAUSED c=20
+#: ultralcd.cpp:907
+msgid "Print paused"
+msgstr "Utskriften pausad"
+
+# MSG_RESUME_NOZZLE_TEMP c=20 r=4
+#: mmu.cpp:726
+msgid "Press the knob to resume nozzle temperature."
+msgstr "Tryck pa knappen f\xefr att aterst\xe1lla munstyckstemperaturen."
+
+# MSG_FOLLOW_CALIBRATION_FLOW c=20 r=8
+#: messages.c:46
+msgid "Printer has not been calibrated yet. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Skrivaren har inte kalibrerats \xe1nnu. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
+
+# MSG_PRINT_FAN c=10
+#: ultralcd.cpp:1447
+msgid "Print FAN"
+msgstr "Utskrfl\xe1kt"
+
+# MSG_WIZARD_LOAD_FILAMENT c=20 r=6
+#: ultralcd.cpp:4818
+msgid "Please insert filament into the extruder, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i extrudern, tryck sedan pa knappen f\xefr inladdning.."
+
+# MSG_MMU_INSERT_FILAMENT_FIRST_TUBE c=20 r=6
+#: ultralcd.cpp:4813
+msgid "Please insert filament into the first tube of the MMU, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i den f\xefrsta kanalen pa MMU:n och tryck sedan pa knappen f\xefr inladdning."
+
+# MSG_PLEASE_LOAD_PLA c=20 r=4
+#: ultralcd.cpp:4735
+msgid "Please load filament first."
+msgstr "V\xe1nligen ladda filament f\xefrst."
+
+# MSG_BED_CORRECTION_REAR c=14
+#: ultralcd.cpp:3117
+msgid "Rear side [um]"
+msgstr "Baksida [um]"
+
+# MSG_UNLOAD_FILAMENT_REPEAT c=20 r=4
+#: ultralcd.cpp:7404
+msgid "Please unload the filament first, then repeat this action."
+msgstr "V\xe1nligen mata ut filamentet f\xefrst och upprepa sedan denna atg\xe1rd."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.